Formula/gtk+3.rb

```diff
@@ -33,15 +33,28 @@ class Gtkx3 < Formula
 
   uses_from_macos "libxslt" => :build # for xsltproc
 
+  on_linux do
+    depends_on "cmake"
```

Comment:
Not a build dependency?

Reply:
Yeah, this should only be a build dependency.

Reply:
I also still find it bizarre that you can build with `meson` but somehow still need `cmake`. But I know it can happen...